Kernel/Arch/aarch64/RegisterState.h

```diff
@@ -22,19 +22,17 @@ struct RegisterState {
     u64 tpidr_el0; // EL0 thread ID
```

Comment:
Again the commit message (b5f36d2c09cd31de956af73994f541f7b9691fcc) does not describe what is actually changed. In any case, the word `link` does not make any sense here.

Reply:
The commit message still does not describe all the changes, copying the register state in fork is one of the changes part of this commit, however the other part that implements various function in RegisterState.h and ThreadRegisters.h should be a different commit, or at least should be mentioned in the commit message.